pkg/skaffold/build/ko/schema/temporary.go

```diff
@@ -0,0 +1,84 @@
+/*
+Copyright 2021 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Temporary home for schema changes.
+
+package schema
+
+// KoArtifact builds images using [ko](https://github.com/google/ko).
+type KoArtifact struct {
+	// BaseImage overrides the default ko base image.
+	// Corresponds to, and overrides, the `defaultBaseImage` in `.ko.yaml`.
```

Comment:
What is the default if unspecified?

Reply:
That'll be the ko defautl of `gcr.io/distroless/static:nonroot`.

I've been thinking that we can default to `gcr.io/distroless/base:debug-nonroot` if `RunMode` is debug.

Reply:
Added comment to document the default base image.